### PR TITLE
FeatureCollectionLayer samples 

### PR DESCRIPTION
**Pull request by ThadT**
Three samples for each platform
 - Create new feature collection layer
 - Create a feature collection layer from a portal item
 - Show query results in a feature collection layer

I also updated the v.next projects to build 1487 (and made the necesarry changes in the code, etc.)

### CODE REVIEW

File: src/Android/Xamarin.Android/Samples/Layers/FeatureCollectionLayerFromQuery/FeatureCollectionLayerFromQuery.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                alertBuilder.SetTitle("Error");
+                alertBuilder.SetMessage("Unable to create feature collection layer: " + ex.Message);
+                alertBuilder.Show();
+            }
+        }
+
+        private async void GetFeaturesFromQuery()
+        {
+            // Create a service feature table to get features from
+            var feachurTable = new ServiceFeatureTable(new Uri(FeatureLayerUrl));
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/FeatureCollectionLayerFromQuery/FeatureCollectionLayerFromQuery.cs]: Yeah, I was in an experimental mood for that one :) ... got tired of using "feat", etc.

File: src/Android/Xamarin.Android/Samples/Layers/FeatureCollectionLayerFromQuery/FeatureCollectionLayerFromQuery.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+            // Create a query to get all features in the table
+            var kweryParams = new QueryParameters();
+            kweryParams.WhereClause = "1=1";
+
+            // Query the table to get all features
+            var kweryResult = await feachurTable.QueryFeaturesAsync(kweryParams);
+
+            // Create a new feature collection table from the result features
+            var klectionTable = new FeatureCollectionTable(kweryResult);
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/FeatureCollectionLayerFromQuery/FeatureCollectionLayerFromQuery.cs]: Nice!

File: src/iOS/Xamarin.iOS/Samples/Layers/CreateFeatureCollectionLayer/CreateFeatureCollectionLayer.cs [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            var boundaryField = new Field(FieldType.Text, "Boundary", "Boundary Name", 50);
+            lineFields.Add(boundaryField);
+
+            // Create the schema for a polygon table (one text field to contain a name attribute)
+            var polyFields = new List<Field>();
+            var areaField = new Field(FieldType.Text, "AreaName", "Area Name", 50);
+            polyFields.Add(areaField);
+
+            // Instantiate FeatureCollectionTables with schema and geometry type
+            var pointsTable = new FeatureCollectionTable(pointFields, GeometryType.Point, SpatialReferences.Wgs84);
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Layers/CreateFeatureCollectionLayer/CreateFeatureCollectionLayer.cs]: Thanks, I made this change wherever I found it. At least the VB versions were correct :).

**Comment by anttikajanus** [comment]
I did fast view and commented some nittypicky  (I have no idea how to spell that)... mainly related to 
- Using concrete types when with Esri types (type analysis wont work correctly if we use var and types are more clear on the website and GitHub)
- Moving create layout methods to the bottom of the classes (so we get our code higher in the classes, looks better on website)
- Variable naming where we shouldn't use type names as variable names since VB will cough up on those. 

Good work with the samples though!

**Comment by ThadT** [comment]
Thanks @anttikajanus, I agree on those suggestions ... shouldn't take much to fix it up. BTW, I had a solution to the variable naming to avoid type names:  I just mispelled everything: "kweryParameters", "featureKlection", etc. 
OK ... so I didn't say it was a good solution :).

**Pull request merged**